sessions/introduction.qmd

```diff
@@ -88,71 +95,72 @@ Why do we structure it this way?
     `data-raw/` folder, R scripts/functions in the `R/` folder, and
     documents like R Markdown / Quarto files in `doc/`.
 
-This also supports our workflow and processes, which will be something
-like:
+This structure also supports our workflow and processes throughout the
```

Comment:
There's something about this overview of the workflow that isn't entirely clear to me. I'll think more about this as I go through the rest of the course material -- I want to make the diagram showing the workflow 👍